2024/waypoints/waypoint_task_1.csv

```diff
@@ -1,2 +1,5 @@
 name,latitude,longitude
-Alpha,48.5112750,-71.6505486
\ No newline at end of file
+Takeoff,48.5102184,-71.6473002
+Alpha,48.5112750,-71.6505486
+Bravo,48.5058636,-71.6322229
+Landing,48.5103379,-71.6472006
```

Comment:
Can we get rid of these changes. We only need to know the waypoint Alpha because when the drone takes off, it will need to fly to waypoint Alpha, then run laps.